openpype/hosts/maya/plugins/publish/validate_render_image_rule.py

```diff
@@ -23,7 +23,13 @@ class ValidateRenderImageRule(pyblish.api.InstancePlugin):
 
     def process(self, instance):
 
-        assert get_file_rule("images") == "renders", (
+        default_render_file = instance.context.data.get('project_settings')\
+            .get('maya') \
+            .get('create') \
+            .get('CreateRender') \
+            .get('default_render_image_folder')
+
+        assert get_file_rule("images") == default_render_file, (
             "Workspace's `images` file rule must be set to: renders"
```

Comment:
Should probably modify the error message as well.

Reply:
Yes, it's done